src/main/java/com/breskul/bibernate/persistence/GenericDao.java

```diff
@@ -84,10 +83,6 @@ public <T> T findById(Class<T> cls, Object id) {
     Field idField = findEntityIdField(cls);
     String idColumnName = resolveColumnName(idField);
     checkEntityIdType(cls, id);
-    T cachedEntity = context.getEntity(cls, id);
-    if (cachedEntity != null) {
-      return cachedEntity;
-    }
```

Comment:
Do we really don't need to get here entity from cache?

Reply:
it done on session level